content/docs/rendering-elements.md

```diff
@@ -1,75 +1,77 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: رندر شدن المان‌ها
```

Comment:
```suggestion
title: رندر کردن المان‌ها
```

Reply:
@sorousht @zahrajoulaei How about using "المنت" instead of "المان"?
They look so familiar
My reasoning is getting muscle memory of the reader used to hearing "element" word pronunciation. "المان" is still a translation and needs some brain lookup when reading some Persian piece and translating it to "element" or vice versa.
My intention is to eliminate this very small brain work.
What do you think?

Reply:
<p dir="ltr">
</p>
@sJJdGG Totally agree with the idea of using familiar technical words to make it more closer to our daily technical talks as much as possible. on the other hand, finding an appropriate farsi word is my concern. I was reading the text once again and found that using "المنت" feels better. So, I personally vote for "المنت". I will fix it if @zahrajoulaei is on the same page.

Reply:
@sJJdGG I applied your requested changed. Thanks for taking care of these issues precisely.

Reply:
@sorousht No problem. Just doing my part.